Geometry/VeryForwardGeometry/python/geometryRPFromDB_cfi.py

```diff
@@ -1,12 +1,21 @@
 import FWCore.ParameterSet.Config as cms
+from Geometry.VeryForwardGeometryBuilder.CTPPSGeometryESModule_cfi import CTPPSGeometryESModule
 
 XMLIdealGeometryESSource_CTPPS = cms.ESProducer("XMLIdealGeometryESProducer",
                                                 rootDDName = cms.string('cms:CMSE'),
                                                 label = cms.string('CTPPS'),
                                                 appendToDataLabel = cms.string('XMLIdealGeometryESSource_CTPPS')
                                                 )
 
-ctppsGeometryESModule = cms.ESProducer("CTPPSGeometryESModule",
-    verbosity = cms.untracked.uint32(1),
+ctppsGeometryESModule = CTPPSGeometryESModule.clone(
     compactViewTag = cms.string('XMLIdealGeometryESSource_CTPPS')
 )
+
+from Configuration.Eras.Modifier_ctpps_2016_cff import ctpps_2016
+ctpps_2016.toModify(ctppsGeometryESModule, isRun2=cms.bool(True))
```

Comment:
It would be better to not add explicit `cms.bool()` here (in https://github.com/cms-sw/cmssw/commit/9b5a389cd3c5d23ab0e70a555cff2841b0cbab03)
```suggestion
ctpps_2016.toModify(ctppsGeometryESModule, isRun2=True)
```

Also, all the three could be combined along
```py
(ctpps_2016 | ctpps_2017 | ctpps_2018).toModify(ctppsGeometryESModule, isRun2=True)
```

Reply:
Many thanks, @makortel ! Committed your proposal in cba5238.